src/js/profile/mobile/widget/Calendar.js

```diff
@@ -160,7 +160,7 @@
 				var self = this,
 					ui = self._ui,
 					firstDate = new Date(self._todayYear, self._todayMonth - 1, 0),
-					lastDate = new Date(self._todayYear, self._todayMonth, 0),
+					lastDate = new Date(self._todayYear, self._todayMonth - 1, 0),
```

Comment:
This is little strange. Now, firstDate and lastDate are the same.

Reply:
ok. I remove it. and change exist date value.